src/main/java/edu/ucla/library/lambda/kakadu/converter/KakaduConverter.java

```diff
@@ -40,18 +48,27 @@
     /* A S3 client for the test to use */
     private AmazonS3 myS3Client = AmazonS3ClientBuilder.standard().build();
 
+    /**
+     * Endpoint where something that monitors this process lives. It's expected to receive PATCH updates and to have
```

Comment:
this is really minor, but the have on this line is duplicated in the next, one of them should be removed

Reply:
Updated comment, thanks.